src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMan = people.filter(person => person.sex === 'm');
+
+  const filterManCentury = filterMan
+    .filter(person =>
+      Math.ceil(person.died / 100) === century);
```

Comment:
```suggestion
  const filteredMen = people.filter(({ sex, died }) => century 
    ? sex === 'm' && Math.ceil(died / 100) === century
    : sex === 'm',
  );
```
consider to decrease amount of code and make code more readable.
check century inside filter callback and in reduce method will be no need ternary.
use destructuring.